pkg/server/server.go

```diff
@@ -334,23 +335,21 @@ func (s *ImmuServer) loadSystemDatabase(dataDir string, adminPassword string) er
 
 	_, sysDbErr := s.OS.Stat(systemDbRootDir)
 	if s.OS.IsNotExist(sysDbErr) {
-		if s.Options.GetAuth() {
-			db, err := database.NewDb(op, nil, s.Logger)
-			if err != nil {
-				return err
-			}
+		db, err := database.NewDb(op, nil, s.Logger)
+		if err != nil {
+			return err
+		}
 
-			s.sysDb = db
-			//sys admin can have an empty array of databases as it has full access
-			adminUsername, _, err := s.insertNewUser([]byte(auth.SysAdminUsername), []byte(adminPassword), auth.PermissionSysAdmin, "*", false, "")
-			if err != nil {
-				return logErr(s.Logger, "%v", err)
-			}
+		s.sysDb = db
+		//sys admin can have an empty array of databases as it has full access
+		adminUsername, _, err := s.insertNewUser([]byte(auth.SysAdminUsername), []byte(adminPassword), auth.PermissionSysAdmin, "*", false, "")
+		if err != nil {
+			return logErr(s.Logger, "%v", err)
+		}
 
-			time.Sleep(time.Duration(10) * time.Millisecond)
+		time.Sleep(time.Duration(10) * time.Millisecond)
```

Comment:
I think is not needed anymore since by default wait for indexing is true

Reply:
I did not touch that part, seems unrelated to the scope of the PR.
But of course, i can delete that line if we are sure it is useless and won't affect anything.

Reply:
Done.